portfolio/src/main/webapp/pages/chat.html

```diff
@@ -0,0 +1,82 @@
+        
+        <!-- Core theme CSS (includes Bootstrap)-->
+        <link href="../css/styles.css" rel="stylesheet" />
+    </head>
+
+    <body id="page-top" onload="loadTasks()">
+
+        <!-- Navigation-->
+        <nav class="navbar navbar-expand-lg navbar-dark fixed-top" id="mainNav">
+            <div class="container">
+                <a class="navbar-brand js-scroll-trigger" href="#page-top">
+                    <img src="../assets/img/Symb.svg" alt="" />
+                </a>
+                <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">Menu<i class="fas fa-bars ml-1"></i></button>
+                <div class="collapse navbar-collapse" id="navbarResponsive">
+                    <ul class="navbar-nav text-uppercase ml-auto">
+                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="../index.html">Home</a></li>
+                        <!-- Optional
+                        <li class="nav-item"><a class="nav-link js-scroll-trigger" href="#contact">Contacts</a></li>
+                        -->
+                    </ul>
+                </div>
+            </div>
+        </nav>
+
+        <section class="page-section" id="comments">
+            <div class="container">
+                <div class="text-center">
+                    <h1><a href="/data">Test</a></h1>
+                    <ul id="chat"></ul>
+                </div>
+                <div class="text-center">
+                    <form action="/data" method="POST"> 
+                        <p>Your comment</p>
+                        <input type="text" placeholder="Enter your name" name="nameBox">
+                        <br/>
+                        <textarea name="chatbox">Comment...</textarea>
+                        <br/><br/>
+
+                        <input type="submit" />
+                    </form>
+                    <ul id="comments"></ul>
+                </div>
+            </div>
+        </section>
+
+        <!-- Bootstrap core JS-->
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.bundle.min.js"></script>
+        <!-- Third party plugin JS-->
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery-easing/1.4.1/jquery.easing.min.js"></script>
+        
+        <!-- Contact form JS-->
+        <!-- Add in case contacts need be added
+        <script src="assets/mail/jqBootstrapValidation.js"></script>
+        <script src="assets/mail/contact_me.js"></script>
+        -->
+        <!-- Core theme JS-->
+
+        <script src="../js/scripts.js"></script>
+        <script src="../js/chatSystem.js"></script>
```

Comment:
Is this a requirement for these scripts to be in the body?

Can't they be in the Head?
Also are all of them needed?